references/detection/run_mask_rcnn_tpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Prerequisite: conda activate torch-xla-nightly-vision
+set -e
+
+# TPU XLA
+export TPU_IP_ADDRESS="10.2.101.2"
+export XRT_TPU_CONFIG="tpu_worker;0;$TPU_IP_ADDRESS:8470"
+
+# CPU XLA
+#export XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0"
+#export XRT_WORKERS="localservice:0;grpc://localhost:40934"
+
+export TF_CPP_VMODULE=tensor=5 # logging
+export TRIM_GRAPH_SIZE=100000
```

Comment:
Are these absolutely needed?

Reply:
Not `TF_CPP_VMODULE` of course. :smiley:
`TRIM_GRAPH_SIZE` is probably not needed either, I added it back when I was running into a lot of XLA issues and I think I either never looked at it or set it to the default value. Let me check what's happening without it.